### PR TITLE
Add placeholder for supervisory hybrid plant controller

### PR DESCRIPTION
**Pull request by misi9170**
This pull request adds a placeholder for a controller for a hybrid wind, solar, and battery plant. Power measurements from each asset are provided to the supervisory controller; and the supervisory controller sends back setpoints. However, there is currently no control logic providing feedback (the setpoints are simply set arbitrarily).

TODO:
- [X] Open PR on Hercules to have solar module accept control setpoints (https://github.com/NREL/hercules/pull/107)
- [X] Add example
- [x] Add documentation
- [x] Add tests

### CODE REVIEW

File: whoc/interfaces/hercules_hybrid_actuator_disk_interface.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+    def get_measurements(self, hercules_dict):
+        turbine_powers = (
+            hercules_dict["hercules_comms"]["amr_wind"][self.wind_name]["turbine_powers"]
+        )
+        time = hercules_dict["time"]
+
+        if ("external_signals" in hercules_dict
+            and "plant_power_reference" in hercules_dict["external_signals"]):
+            plant_power_reference = hercules_dict["external_signals"]["wind_power_reference"]
```
Comment (genevievestarke) [inline whoc/interfaces/hercules_hybrid_actuator_disk_interface.py:43]: Does this mean that the plant power reference needs to be defined as "wind_power_reference" always?

Comment (misi9170) [reply whoc/interfaces/hercules_hybrid_actuator_disk_interface.py:43]: @genevievestarke good catch!! That was supposed to be `"plant_power_reference"` (copy and paste error). I've now fixed this

File: docs/controllers.md [hunk 2]
```diff
@@ -48,3 +48,12 @@ Integral action, as well as gain scheduling based on turbine saturation, has bee
 simple proportional control appears sufficient currently. However, these may be enabled at a 
 later date if needed. The `proportional_gain` for the controller may be provided on instantiation,
 and defaults to `proportional_gain = 1`.
+
+### HybridSupervisoryControllerBaseline
```
Comment (misi9170) [inline docs/controllers.md:52]: @genevievestarke could you check the documentation below and see if I've accurately portrayed the `HybridSupervisoryControllerBaseline`?

**Comment by misi9170** [comment]
@genevievestarke would you like the simple_hybrid_plant example to be replaced by your wind_solar_battery one? the wind_solar_battery example is not yet in the repository, but we could add it if you'd like.

**Comment by genevievestarke** [comment]
The simple_hybrid_plant example doesn't work for me. Does it work for you? I was wondering if you want me to put in the example plant I've been running?

**Comment by genevievestarke** [comment]
@misi9170 I cleaned up the jupyter notebook, it should be ready for merging after your review!

**Comment by misi9170** [comment]
Thank @genevievestarke, I'll take a look over it tomorrow and we should be able to get it merged

File: docs/controllers.md (hunk 2 above)
Comment (misi9170) [reply docs/controllers.md:52]: @genevievestarke , just bumping this, are you OK with the description I've written for the `HybridSupervisoryControllerBaseline`? Feel free to update the description if you'd like.

**Comment by misi9170** [comment]
@genevievestarke I made a few minor cosmetic changes to the example notebook, but I think this is good to merge. As I opened this PR, I can't review it, so if you'd like to add a review feel free.

If you get a chance, have a look over the controller description I added; assuming that's fine, I think we're good to merge.

**Comment by genevievestarke** [comment]
Ok, I updated the controller description, should be good to merge!

**Review by genevievestarke — APPROVED** [review]

**Pull request merged**